packages/fxa-payments-server/src/components/PaymentFormV2/index.tsx

```diff
@@ -122,10 +122,21 @@ export const PaymentForm = ({
     [validator, onSubmitForParent, stripe, submitNonce, shouldAllowSubmit]
   );
 
-  const { matchMedia, navigatorLanguages } = useContext(AppContext);
-  const stripeElementStyles = mkStripeElementStyles(
-    matchMedia(SMALL_DEVICE_RULE)
-  );
+  const { navigatorLanguages } = useContext(AppContext);
+
+  const STRIPE_ELEMENT_STYLES = {
```

Comment:
We don't actually need any kind of media type logic here (we can just uniformly set these rules with the new form.